src/include/pgmoneta.h

```diff
@@ -198,6 +198,7 @@ struct server
    char wal_slot[MISC_LENGTH];         /**< The WAL slot name */
    char current_wal_filename[MISC_LENGTH]; /**< The current WAL filename*/
    char follow[MISC_LENGTH];           /**< Follow a server */
+   char pgmoneta_current_wal_lsn[MISC_LENGTH]; /**< The current WAL log sequence number*/
```

Comment:
Move it next to `current_wal_filename` and rename to `current_wal_lsn`

Reply:
I'll push the code with the update